joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/ControlStructureTests.scala

```diff
@@ -887,23 +899,35 @@ class ControlStructureTests extends PhpCode2CpgFixture {
         inside(matchStructure.astChildren.collectAll[Block].astChildren.l) {
           case List(
                 aTarget: JumpTarget,
+                aCond: Identifier,
                 aValue: Literal,
                 bTarget: JumpTarget,
+                bCond: Identifier,
                 cTarget: JumpTarget,
+                cCond: Identifier,
                 otherValue: Literal
               ) =>
             aTarget.code shouldBe "case $a"
             aTarget.lineNumber shouldBe Some(3)
 
+            aCond.code shouldBe "$a"
+            aCond.lineNumber shouldBe Some(3)
+
             aValue.code shouldBe "\"A\""
             aValue.lineNumber shouldBe Some(3)
 
             bTarget.code shouldBe "case $b"
             bTarget.lineNumber shouldBe Some(4)
 
+            bCond.code shouldBe "$b"
+            bCond.lineNumber shouldBe Some(4)
+
             cTarget.code shouldBe "case $c"
             cTarget.lineNumber shouldBe Some(4)
 
+            cCond.code shouldBe "$c"
+            cCond.lineNumber shouldBe Some(4)
+
```

Comment:
For a case like 
```
match (...) {
  $b, $c => ...
}
```
I'm not sure if we want separate jump targets for the 2 variables or one shared. Having the separate targets creates a clearer link between jump target and condition, in my opinion.